src/content/deployment/_index.en.md

```diff
@@ -52,5 +52,7 @@ For each cluster you want to join, issue the following command:
 subctl join --kubeconfig <PATH-TO-JOINING-CLUSTER> broker-info.subm --clusterid <ID>
 ```
 
-subctl will automatically discover as much as it can, and prompt the user for any missing necessary information. Note that each cluster must
-be configured with a unique cluster ID.
+subctl will automatically discover as much as it can, and prompt the user for
```

Comment:
The max line length enforced by all linters for all types of files is 140 chars, so imho it's best to wrap at that length or wrap early if you can finish a sentence (to make line-by-line diffs nicer, as changes are typically sentence-based).